interpreter/cling/include/cling/std.modulemap

```diff
@@ -309,7 +309,7 @@ module "std" [system] {
     header "string"
   }
   module "string_view" {
-    requires cplusplus17, !header_existence
```

Comment:
Can you explicitly revert my commit? I find that cleaner than adding this into your commit.